.github/workflows/tutorials.yml

```diff
@@ -0,0 +1,52 @@
+name: Jupyter notebooks
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: Test Jupyter notebooks on ${{ matrix.python-version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: ['3.6', '3.7']
+        os: [ubuntu-latest]
+        gcc-version: [9]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install GCC ${{ matrix.gcc-version }}
+      run : |
+        sudo apt-get install -y g++-${{ matrix.gcc-version }}
+
+    - name: Install MPI
+      run : |
+        sudo apt-get install -y mpich libmpich-dev
+
+    - name: Install dependencies
+      run: |
+        pip install --upgrade pip
+        pip install -r requirements.txt
+        pip install -r requirements-optional.txt
+        pip install -e .[extras]
+
+    - name: Evaluate cells of example notebooks
+      env:
+        CC: ${{ matrix.gcc-version }}
+      run: |
+        for notebook in examples/cfd examples/seismic/tutorials examples/compiler examples/userapi
```

Comment:
As I can see from the log, it is not testing all the targetted notebooks this way, right?

Reply:
Could discuss more on what you mean here. 

When look at the `.travis.yml` file, the only notebooks being tested are in the above folder. From the logs it seems they are indeed being tested via actions too.

Reply:
Resolving conversation for now as at this time, it is outdated. Happy to reopen if preferred